app/templates/press-releases/press-release/edit.hbs

```diff
@@ -0,0 +1,130 @@
+<Auk::Navbar @skin="gray-100" @size="large">
+  <Auk::Toolbar>
+    <Auk::Toolbar::Group  @position="left">
+      <Auk::Toolbar::Item>
+        <Auk::Button
+          @layout="icon-only"
+          @skin="borderless"
+          @icon="chevron-left"
+          @size="large"
+          {{on "click" (perform this.navigateBack)}}
```

Comment:
Recent van @MikiDi geleerd: ipv in deze actie een eigen logica te bouwen met `this.fromRoute` kunnen we beter gebruik maken van [de History API van de browser](https://developer.mozilla.org/en-US/docs/Web/API/History/back) om terug te navigeren. Zie vb. in Kaleidos: https://github.com/kanselarij-vlaanderen/kaleidos-frontend/blob/eb5bc9b6363552b915bd99216ad65c1cf79f7cdd/app/pods/document/controller.js#L11-L15